### PR TITLE
upgrade to django 1.8

### PR DESCRIPTION
**Pull request by muhammad-ammar**
TNL-2824

### CODE REVIEW

**Comment by muhammad-ammar** [comment]
@wedaly @symbolist Please have a final look. Thanks!

File: .travis.yml [hunk 1]
```diff
@@ -10,5 +10,6 @@ script:
 branches:
     only:
       - master
+      - will/upgrade-django-rest-framework
```
Comment (wedaly) [inline .travis.yml]: This has been merged to master, so I don't think we need this anymore.

**Comment by wedaly** [comment]
One small comment, otherwise :+1:

**Comment by muhammad-ammar** [comment]
@wedaly ah...great catch..thanks

**Comment by macdiesel** [comment]
We need to update the version number on this library as well.

**Pull request merged**